apps/src/templates/teacherNavigation/teacher-navigation.module.scss

```diff
@@ -9,13 +9,13 @@
 
 .pageWithHeader {
   flex-grow: 1;
-  padding: 20px;
+  padding: 20px 48px 20px 38px;
   overflow-y: auto;
 }
 
 .widthLockedPage {
   max-width: $content-width;
-  padding: 10px 64px 64px 64px;
+  padding: 10px 64px 64px 0;
```

Comment:
What do you think about removing the right padding as well? Currently these pages have 48px from the pageWithHeader and 64px from widthLockedPage